src/layout-rect.js

```diff
@@ -148,6 +152,8 @@ export function expandLayoutRect(rect, dw, dh) {
     right: rect.right + rect.width * dw,
     width: rect.width * (1 + dw * 2),
     height: rect.height * (1 + dh * 2),
+    x: rect.x - rect.width * dw,
```

Comment:
We could probably call `layoutRectLtwh` with the appropriate values (reduces code here, uses same internal class, less math operations)